fast64_internal/oot/__init__.py

```diff
@@ -32,13 +32,14 @@ def draw(self, context):
 
         prop_split(col, context.scene, "ootDecompPath", "Decomp Path")
         col.prop(context.scene.fast64.oot, "hackerFeaturesEnabled")
-
+        col.prop(context.scene.fast64.oot, "shipOfHarkinianCompatible")
```

Comment:
I'm confused why you are calling it this considering that as far as I can tell this feature does not merely allow "compatibility" with the PC port, but rather it changes the output such that it is no longer compatible with N64. As an aside, I am a little confused why such a PR is really necessary when the PC port tools should already be able to handle converting N64 models to whatever format it is expecting. Can you explain why it is necessary that Fast64 be modified to support the PC port?